src/Vlingo.Xoom/vlingo-xoom.json

```diff
@@ -0,0 +1,29 @@
+﻿{
+  "database": "IN_MEMORY",
+  "database.name": "",
+  "database.driver": "",
+  "database.url": "",
+  "database.username": "",
+  "database.password": "",
+  "database.originator": "",
```

Comment:
This is not the correct structure we discussed about.

It should be
```
{
  "database": {
    "name": "",
    "driver": "",
    // and so one
  },
  "query": {
    "database": {
       "name": "STORAGE_TEST"
       "driver": "...;"
       // and so on
    }
  }
 }
}
```

Reply:
Changed to
**{
  "database": {
    "name": "",
    "driver": "",
    "url": "",
    "username": "",
    "password": "",
    "originator": ""
  },
  "query": {
    "database": {
      "name": "STORAGE_TEST",
      "driver": "",
      "url": "jdbc:mysql://localhost:2215/",
      "username": "vlingo_test",
      "password": "vlingo123",
      "originator": "MAIN"
    }
  },
  "exchange": {
    "names": "first;second",
    "first": {
      "hostname": "first-exchange",
      "username": "vlingo01",
      "password": "vlingo-pass01",
      "port": "1000",
      "virtual": {
        "host": "virtual-first-exchange"
      }
    },
    "second": {
      "hostname": "second-exchange",
      "username": "vlingo02",
      "password": "vlingo-pass02",
      "port": "1001",
      "virtual": {
        "host": "virtual-second-exchange"
      }
    }
  }
}**

Reply:
do i need to remove **"url": "jdbc:mysql://localhost:2215/",** too?